src/Scrabble/Move/Move.hs

```diff
@@ -42,9 +42,13 @@ createMove' :: Validator -- ^
 createMove' validate b (Rack rack) wp dict = if valid then go else errMsg where
   errMsg        = Left "error: rack missing input letters"
   rackLetters   = fmap letter rack
-  valid         = containsAll (toString putLetters) (toString rackLetters)
+  valid         = containsAllWithBlanks (toString putLetters) (toString rackLetters)
+  blanksUsed    = countBlanksUsed (toString putLetters) (toString rackLetters)
   putLetters    = letter <$> wordPutTiles wp
-  rackRemainder = fmap fromLetter $ foldl' (flip delete) rackLetters putLetters
+  rackRemainder = fmap fromLetter $
+                  applyNTimes blanksUsed (delete Blank) $
+                  foldl' (flip delete) rackLetters putLetters
+  applyNTimes n f = foldr (.) id (replicate n f)
```

Comment:
That's what I was going for initially, but I arrived here on my way to that and figured I'd get your input. It's kind of an accidental "keep you from shooting yourself in the foot" feature